sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala

```diff
@@ -343,3 +343,31 @@ class ContinuousMetaSuite extends ContinuousSuiteBase {
     }
   }
 }
+
+class ContinuousEpochBacklogSuite extends ContinuousSuiteBase {
+  import testImplicits._
+
+  // We need to specify spark.sql.streaming.continuous.epochBacklogQueueSize.
```

Comment:
We should be able to set the queue size conf using withSQLConf; see examples in e.g. ContinuousAggregationSuite.

I'd suggest a comment explaining that this test forces the backlog to overflow by not standing up enough executors for the query to make progress. (Probably we should make that an immediate error at some point, which means we'd need to come up with another way to test this, but I think the current strategy is good for now.)

Reply:
Switched to `withSQLConf` and added a comment.